assets/index.js

```diff
@@ -0,0 +1,138 @@
+//Index.js
```

Comment:
```suggestion
// @ts-check

//Index.js
```

With this comment you can get type validation for free in VS Code in JS files (without having to write TS)